IMC_WebDev/FinalBlog.md

```diff
@@ -0,0 +1,224 @@
+5. Transform: This property is used to make 2D transformations. The most commonly used transformations are rotate() and skew.
+* rotate(): This transformation rotates an element clockwise or counter-clockwise according to a given degree. For example, 
+transform: rotate(20deg).
+* skew(): Unlike rotate, this transformation skews an element along the X and Y-axis by the given angles. For example, this example skews the element 30 degrees along the X-axis and 50 degrees along the Y-axis.
+transform: skew(30deg, 50deg);
+6. Top: This affects the vertical position of the element. The above example sets the element 18px below the top edge of its nearest positioned ancestor.
+7. Left: In contrast, this affects the horizontal position of the element. The above example sets the element 90px from the left edge of its nearest positioned ancestor.
+8. Z-Index: This property specifies the stack order of an element. You can think about Z-Index like layers of a sandwich. If the Z-Index value is higher for one item than another, the higher value will be drawn first. 
+
+To begin with, you will need a foundation for your city to be built on. You can use the following code as a starting off point:
+```html
+.ground{ /*foundation*/
+        position: absolute;
+        display: block;
+        /*purple*/
+        background-color: #D3D3D3;
+        width: 200px;
+        height: 148px;
+        transform: skew(60deg, -15deg);
+        top: 100px;
+        left: -150px;
+        z-index:-999;
+      }
+      .ground .bottomSide{
+        position: absolute;
+        display: block;
+        width: 16px;
+        height: 148px;
+        background-color: #C0C0C0;
+        transform: skew(0deg, -30deg);
+        top: 4px;
+        left: -16px;
+        z-index:-1001;
+      }
+      .ground .bottomFront{
+        position: absolute;
+        display: block;
+        width: 210px;
+        height: 9.5px;
+        background-color: #C0C0C0;
+        transform: skew(-60deg, 0deg);
+        top: 147px;
+        left: -8px;
+        z-index:-1001;
+      }
+```
+
+Now that you have your foundation, you might want to start building. To help you get started, this is what the code to draw a road looks like:
+```html
+.ground .road1 {
+      	position: absolute;
+        display: block;
+        background-color: #C0C0C0;
+        width: 20px;
+        height: 200px;
+        transform: rotateZ(90deg);
+        top: 18px;
+        left: 90px;
+        z-index:-998;
+      }
+```
+
+
+Now that you’ve laid down the foundation and built a road, you can start building more roads, bridges, buildings, or anything else you might want in your mini city! This is what our city looked like after creating all the code described in this tutorial: <img src=https://lh4.googleusercontent.com/yu268ztRROLzh3AjLVUeNroSMUlbOsHpZzsWNIokeRtjF7iz-1mqWCqJPdtL1-JyUX6shb3kq28_CYm1xihsf70nPM8EBfHVERg13x7ef5nf5TibItYGI7ixBu_CHcgyIUfRML_c>
```

Comment:
Would add a sentence or two more here, wrapping up about the importance of HTML/CSS to better mirror your introduction.